packages/app-mobile/components/Dropdown.js

```diff
@@ -87,6 +86,7 @@ class Dropdown extends React.Component {
 		if (this.props.labelTransform && this.props.labelTransform === 'trim') headerLabel = headerLabel.trim();
 
 		const closeList = () => {
+			this.props.hasOwnProperty('hideShowUndoRedoButton') && this.props.hideShowUndoRedoButton();
```

Comment:
This is a generic dropdown component, why should it be concerned with hiding or showing unrelated buttons? Instead the component should expose an onOpen and onClose handlers, which the parent component will use to do some action, in this case hiding and showing the buttons.

Reply:
Ok I will do the same